pkg/config/sort.go

```diff
@@ -40,6 +40,18 @@ func SortConfig(cfg *Config) {
 		}
 		sort.Strings(team.Members)
 
+		// Remove and sort and duplicated team mentors
+		teamMentors := make(map[string]struct{}, len(team.Mentors))
+		for _, teamMentor := range team.Mentors {
+			teamMentors[teamMentor] = struct{}{}
+		}
+
+		team.Mentors = make([]string, 0, len(teamMentors))
+		for teamMentor := range teamMentors {
+			team.Mentors = append(team.Mentors, teamMentor)
+		}
+		sort.Strings(team.Members)
```

Comment:
Two things: Why not sort inline? Also the slice being sorted here is `team.Members` not `team.Mentors`.

```suggestion
		sort.Strings(team.Mentors)
```

Reply:
i definitely need to sort team members as well as members.

The code isn't just sorting its de-duping. I'm following the pattern established by the existing sort logic for team members.

Reply:
Oh yeah de-duping makes sense.